doc/dash/dash-sonic-hld.md

```diff
@@ -696,10 +711,17 @@ key                      = DASH_TUNNEL_TABLE:tunnel_name; tunnel name used for r
 ; field                  = value
 endpoints                = list of addresses for ecmp tunnel
 encap_type               = vxlan or nvgre
-vni                      = vni value for encap
+vni                      = vni value for encap, create only attribute
```

Comment:
should we mention create only attribute for encap_type as well ?

Reply:
addressed